projects/packages/backup/src/js/components/Admin.js

```diff
@@ -111,6 +171,27 @@ const Admin = () => {
 		);
 	};
 
+	const renderReviewMessage = () => {
```

Comment:
Instead of using a function/call to `renderReviewMessage`,define it like `const ReviewMessage = ` and use it like `<ReviewMessage/>` from the `<Admin/>` component. This way you get to see it as a component with the React dev tools.

Reply:
Try to do the same for every piece of `renderXXX` that is defined in this file.